### PR TITLE
[stable/falco]: Restrict the access to /dev on underlying host to rea…

### PR DESCRIPTION
**Pull request by faarshad**
#### What this PR does / why we need it:
This PR improves the security posture of falco container itself by only allowing readOnly access to `/dev` on underlying host. Following the [least-privilage access principle](https://en.wikipedia.org/wiki/Principle_of_least_privilege) is a nice to have, especially for a security tool itself.

#### Special notes for your reviewer:
I tested this change locally on minikube. Falco container was able to correctly load kernel modules and start cleanly. I also tested the detection by falco when a privileged container is spawned.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [X] Chart Version bumped
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by bencer** [comment]
I would feel more comfortable getting this merged into upstream first.

https://github.com/falcosecurity/falco/blob/dev/integrations/k8s-using-daemonset/k8s-with-rbac/falco-daemonset-configmap.yaml#L34

**Comment by farshad-hobsons** [comment]
> I would feel more comfortable getting this merged into upstream first.
> 
> https://github.com/falcosecurity/falco/blob/dev/integrations/k8s-using-daemonset/k8s-with-rbac/falco-daemonset-configmap.yaml#L34

Upstream PR https://github.com/falcosecurity/falco/pull/834

**Comment by farshad-hobsons** [comment]
@bencer - fyi - upstream is merged

File: stable/falco/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 This file documents all notable changes to Sysdig Falco Helm Chart. The release
 numbering uses [semantic versioning](http://semver.org).
+## v1.0.8
```
Comment (nestorsalceda) [inline stable/falco/CHANGELOG.md]: Add a whiteline here, on top of version number.

**Review by nestorsalceda — CHANGES_REQUESTED** [review]

**Comment by farshad-hobsons** [comment]
Apologies @nestorsalceda & @ryanaross - Since there were new falco-chart changes since my original changes/pr, I did a rebase and in the process corrupted this PR. I will close this PR and open a new one.

**Pull request closed without merging**